app.js

```diff
@@ -22,6 +23,16 @@ app.use('/scripts', express.static(path.join(__dirname, 'scripts')));
 app.use('/stylesheets', express.static(path.join(__dirname, 'stylesheets')));
 app.use('/lib', express.static(path.join(__dirname, 'lib')));
 
+//Handeling GET and POST request in Express
+app.post('/addUser',(request,response) => {
+    //code to perform particular action.
```

Comment:
rather than using query parameters, it is idiomatic to use the body.
From https://google.aip.dev/133:

> There must be a body key in the google.api.http annotation, and it must map to the resource field in the request message.

https://expressjs.com/en/4x/api.html#req.body